<?xml version="1.0"?>
<launch>
	
  <arg name="topic_name" default="/phantom/pose"/>
  <arg name="write_ip" default="127.0.0.1"/>
  <arg name="write_port" default="34900"/>
  <arg name="publish_rate" default="1000.0"/>
  
  
  <!-- Send UDP Node -->
  <node name="send_udp_pose" pkg="teleop_strategies_controller" type="send_udp_pose.py" output="screen">
	<param name="topic_name" type="string" value="$(arg topic_name)"/>
    <param name="write_ip" type="string" value="$(arg write_ip)"/>
    <param name="write_port" type="int" value="$(arg write_port)"/>
    <param name="publish_rate" type="double" value="$(arg publish_rate)" />
  </node>
  
</launch>
